src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -414,13 +414,7 @@ public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] strin
 
         public static Complex Sin(Complex value)
         {
-            // We need both sinh and cosh of imaginary part. To avoid multiple calls to Math.Exp with the same value,
-            // we compute them both here from a single call to Math.Exp.
-            double p = Math.Exp(value.m_imaginary);
-            double q = 1.0 / p;
-            double sinh = (p - q) * 0.5;
-            double cosh = (p + q) * 0.5;
-            return new Complex(Math.Sin(value.m_real) * cosh, Math.Cos(value.m_real) * sinh);
+            return new Complex(Math.Sin(value.m_real) * Math.Cosh(value.m_imaginary), Math.Cos(value.m_real) * Math.Sinh(value.m_imaginary));
```

Comment:
Just to double check, you plan on opting to use `SinCos` where possible in a separate PR?

-- It might be worth having a `SinhCosh` long term as well, but it's not something that's optimized today.

Reply:
Yes was planning to do the different suggestion a separate PRs, but I can put them all in this one if you prefer?

And yes `SinhCosh` could be useful. It's not very common but seems like a few libraries have that method available.

Reply:
Different PRs is fine by me, just wanted to double check.